src/main/java/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,49 @@
+package duke.commands;
+
+import static java.lang.Integer.parseInt;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+
+
+/**
+ * Class that simulates the done command.
+ */
+
+public class DoneCommand extends Command {
+    private static final String INVALID_INPUT = "Invalid input for done";
+    /**
+     * Creates a DoneCommand object.
+     * 
+     * @param inputArr Array of length 2 that contains information of the user input
+     *                 At index 0, contains the type of command
+     *                 At index 1, contains the message of the command.
+     */
+    public DoneCommand(String[] inputArr) {
+        super(inputArr);
+    }
+
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
+        return marking(parseInt(inputArr[1]), ui, tasks);
+    }
+
+    /**
+     * Mark the task at that index 'pos' of the list. If the index 'pos' less than of equals to 0 or greater than the 
+     * size of the list, a message will printed, notifying the user of the invalid input.
```

Comment:
Hi, some minor grammar mistakes here may make the comment not readable. Perhaps you may correct it.

Reply:
yup thanks